src/main/java/org/generationcp/ibpworkbench/actions/RunMultiSiteAction.java

```diff
@@ -165,13 +165,13 @@ void exportDataFiles(
 		final String inputFileName = this.generateInputFileName(multiSiteParameters.getProject());
 
 		final String meansDataFilePath = this.multiSiteDataExporter
-			.exportMeansDatasetToCsv(FileNameGenerator.generateFileName(inputFileName, "csv", false), multiSiteParameters, this.gxeTable.getExperiments(),
+			.exportMeansDatasetToCsv(inputFileName, multiSiteParameters, this.gxeTable.getExperiments(),
```

Comment:
Is it really correct to completely remove the FileNameGenerator call here?

Reply:
FIlenameGenerator.generateFileName was moved to MultiSiteDataExporter.getCsvFileInWorkbenchDirectory to make sure filename always ends with _username_date_time.csv